app/schemas/org.hackillinois.android.database.Database/2.json

```diff
@@ -0,0 +1,301 @@
+{
```

Comment:
Can you delete 2.json and 3.json, delete the app from your emulator, put Database version to 1, and rerun the app?

Since we aren't actually releasing right now, we don't need to update version numbers. Doing the steps above will regenerate 1.json and let you run the app without the database migration error you were getting earlier.